### DISCUSSION TITLE
How to access images from other app

### DISCUSSION THREAD

**Discussion by kacper-the-programmer** [zepp-health/zeppos-forum#352] [Q&A] [answered]
I'm making an app for Zepp OS 1.0 that's have a lot of images, building this app throue `zeus preview` command takes a lot of time, so I have an idea to split this app in half, by that I mean first ( main ) app would have all the logic, and secound app would have all images that main app need. But is this possible ?

**Comment by kacper-the-programmer** [comment]
I did some experiments and found that when I get an error on the emulator I can get the absolute path in Zepp OS but on mobile I get the file path on the desktop and I found that the apps are stored in a folder called `js apps/<random hash>`, now I'm trying to optimize the brute force algorithm to be able to find the secound app folder and be executable on my GTR 3

**Comment by kacper-the-programmer** [comment, marked as answer]
apparently this hash is the same on every device, so I can get them through emulator

**Reply by SashaCX75** [reply]
I think the correct path would be `js apps/<appId (in hex format)>`

**Reply by kacper-the-programmer** [reply]
oh, your right, I didn't think about it, thanks
